### DISCUSSION TITLE
How to move sent/received messages from Message+ to Fossify messenger.

### DISCUSSION THREAD

**Discussion by Sin-Shadow-Fox** [FossifyOrg/General-Discussion#501] [Q&A]
I tried posting this elsewhere and was told to post it here in messages.

How do i transfer my sent and received messages from Message+ to Fossify as Message+ is shutting down on November 7th. I don’t want to lose several years of messages so i need a solution before the 7th. Please help.

**Comment by Aga-C** [comment]
Have you tried to just install Fossify Messages and check if you have messages? According to https://www.verizon.com/support/verizon-messages-faqs/, for Google Messages it's enough to just install the app. Our app doesn't work this much different.

EDIT: I've found full instructions on how to move your messages: https://www.reddit.com/r/verizon/comments/1d9yle3/comment/l7gpdxo/. Just use Fossify Messages instead of Google Messages.

**Reply by Sin-Shadow-Fox** [reply]
Wow. That was really easy. I've never seen a messenger do that before. I thought i needed to compress them into a file type and transfer it over manually like i did with Messenger to Message+ when i upgraded my Razr 1st gen to my Razr 2nd gen. Thanks ^\\.=./^
